oci/common_columns.go

```diff
@@ -1,11 +1,22 @@
 package oci
 
+import (
+	"context"
+
+	oci_common "github.com/oracle/oci-go-sdk/v65/common"
+	"github.com/oracle/oci-go-sdk/v65/identity"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
 const (
 	// Constants for Standard Column Descriptions
 	ColumnDescriptionAkas        = "Array of globally unique identifier strings (also known as) for the resource."
 	ColumnDescriptionTags        = "A map of tags for the resource."
 	ColumnDescriptionTitle       = "Title of the resource."
 	ColumnDescriptionTenant      = "The OCID of the Tenant in which the resource is located."
```

Comment:
@ParthaI to avoid confusion, I think it would be better to update this constant to `ColumnDescriptionTenantId` and make subsequent updates across all the tables.